### ISSUE TITLE
Allow for self-signed certificates in `morgue setup`

### ISSUE THREAD

**Issue by mbreaux** [backtrace-labs/backtrace-morgue#16]
When using self-signed certificates:

```
$ morgue setup https://localhost
Determining system state...unconfigured
/usr/local/lib/node_modules/backtrace-morgue/node_modules/backtrace-node/lib/index.js:147
    throw err;
    ^

Error: self signed certificate
    at doRequestWith (/usr/local/lib/node_modules/backtrace-morgue/node_modules/sync-request/index.js:87:11)
    at doRequest (/usr/local/lib/node_modules/backtrace-morgue/node_modules/sync-request/index.js:20:10)
    at BPG.refresh (/usr/local/lib/node_modules/backtrace-morgue/dist/lib/bpg.js:127:18)
    at new BPG (/usr/local/lib/node_modules/backtrace-morgue/dist/lib/bpg.js:114:10)
    at coronerSetupStart (/usr/local/lib/node_modules/backtrace-morgue/dist/bin/morgue.js:292:9)
    at /usr/local/lib/node_modules/backtrace-morgue/dist/bin/morgue.js:327:14
    at StreamSink.onFinish (/usr/local/lib/node_modules/backtrace-morgue/dist/lib/coroner.js:101:7)
    at emitNone (events.js:91:20)
    at StreamSink.emit (events.js:185:7)
    at finishMaybe (_stream_writable.js:514:14)
```

Workaround is to use `export NODE_TLS_REJECT_UNAUTHORIZED=0`
It would be better to add an option for self-signed (`-k` ?) or somehow detect this automatically.

**Comment by mbreaux** [comment]
There is a `-k` option, but coronerSetup doesn't check it, but it forces insecure

**Comment by mbreaux** [comment]
Resolved in #17
